stubs/psutil/psutil/_common.pyi

```diff
@@ -263,10 +265,10 @@ class _WrapNumbers:
     def cache_clear(self, name: Any | None = ...) -> None: ...
     def cache_info(self): ...
 
-def wrap_numbers(input_dict, name): ...
+def wrap_numbers(input_dict, name: str): ...
```

Comment:
The docstring is "Given an `input_dict` and a function `name`, ..." so it doesn't look like it is a string.

Reply:
Looks like a typo. I will submit a PR to `psutil`. The only usage of `name` I can find is `str`: https://github.com/giampaolo/psutil/blob/5d39cc9c8d8dc04862786abba0307ed9350144ce/psutil/tests/test_misc.py#L422 and https://github.com/giampaolo/psutil/blob/5d39cc9c8d8dc04862786abba0307ed9350144ce/psutil/__init__.py#L2070